packages/contour/test/src/types/object_test.dart

```diff
@@ -34,13 +34,17 @@ void main() {
       expect(result.value, {'name': 'John', 'age': 30});
     });
 
-    test('parse should return empty fields for non-map fields ', () {
+    test('parse should not return non-mappable fields', () {
       final objectType = ContourObject({
         'name': ContourString(),
         'age': ContourNumber(),
       }).additionalFields(false);
```

Comment:
Perhaps we should give .additionalFields parameters as well just like I'm suggesting for integer, where it would have .additionalFields(force: bool) 

force = true, means strip them out and do not throw an error
force = false, means do a check and throw an error if fields are found

What do you think @ronsilverentand?